### ISSUE TITLE
dual license

### ISSUE THREAD

**Issue by nfriedly** [nfriedly/node-unblocker#41]
Hi @hellais, @emilhem, and @tfMen,

I received a request for an alternate-licensed version of node-unblocker for inclusion in a project that's not compatible with the GPL. (node-unblocker is currently GPL-licensed, which applies to your contributions as well.) Are you guys willing to dual-license your contributions as GPL and MIT?

**Comment by emilhem** [comment]
I am willing to dual license my contributions as GPLv3 and MIT.

**Comment by hellais** [comment]
Good for me :)

I actually have no recollection of having written said code, but I will trust the git history.

**Comment by nfriedly** [comment]
Thanks @emilhem!

Everyone: After thinking about this for a bit, I think I'd actually prefer to keep the main project as GPL and instead offer commercial licenses for users who cannot use GPL software. So, what if I offered each of you a few dollars via paypal or bitcoin or whatever in exchange for assigning copyright of your contributions to me?

According to https://github.com/nfriedly/node-unblocker/graphs/contributors the breakdown of commits is as follows:

Author | Commits	| %
---------|-----------------|-------------
nfriedly	| 146	| 92.99%
emilhem	| 6	| 3.82%
tfMen	| 4	| 2.55%
hellais	| 1	| 0.64%

So, would you each be willing to sell me copyright for your contributions at, say, $10 for each percentage point?

If you don't want to post your details on here, you can send info to nathan@nfriedly.com instead.

**Comment by hellais** [comment]
Sounds good to me.

I assign copyright to Nathan Friedly.

Send any bitcoin donation to the OONI project: https://github.com/TheTorProject/ooni-sysadmin#donate-to-support-ooni-infrastructure or for non bitcoin donations to The Tor Project: https://www.torproject.org/donate/donate.html.en

**Comment by nfriedly** [comment]
I just sent 0.02402374 bitcoins to the OONI project: https://blockchain.info/tx/ff44e068fbf97cfb3efe7ce56b977ccea81f3b83fef2ed91db381aabf0f59973

Thanks @hellais!

**Comment by nfriedly** [comment]
OK, I went and rewrote everything from @tfMen (nothing against his work, but I switched to a different library that covers a few more edge cases, doesn't require compiling, and is reportedly 3-4 times faster), so I think I'm in the clear now for dual-licensing unblocker.

@emilhem I'd still appreciate it if you'd be willing transfer copyright (meaning just reply with something like *"I assign copyright of my node-unblocker contributions to Nathan Friedly."* just because it would make everything a little easier and clear-cut on my end.

Thanks again everyone!
